src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -210,16 +226,21 @@ export default defineConfig({
 });
 ```
 
-### `cloudflareModules`
-
+### sessionKVBindingName
 <p>
-**Type:** `true | false`<br />
-**Default:** `true`
+**Type:** `string`<br />
+**Default:** `SESSION`
 </p>
```

Comment:
```suggestion
<Since v="5.7.0" />
</p>
```
Is this new? It doesn't seem like it was on the page before. Does it need a Since?

Reply:
Ah yes. It's actually since 5.6.0. We just added it.

Reply:
```suggestion
<Since v="5.6.0" />
</p>
```